plugins/modules/cloud/digital_ocean/digital_ocean.py

```diff
@@ -115,17 +115,18 @@
 # If a key matches this name, will return the ssh key id and changed = False
 # If no existing key matches this name, a new key is created, the ssh key id is returned and changed = False
 
-- digital_ocean:
+- name: Ensure a SSH key is present
+  digital_ocean:
     state: present
     command: ssh
     name: my_ssh_key
     ssh_pub_key: 'ssh-rsa AAAA...'
     api_token: XXX
 
-# Create a new Droplet
-# Will return the droplet details including the droplet id (used for idempotence)
-
-- digital_ocean:
+- name: >
+    Create a new Droplet.
+    Will return the droplet details including the droplet id (used for idempotence)
```

Comment:
I would make this a comment. It does not belong in the task name IMO.
```suggestion
  # Will return the droplet details including the droplet id (used for idempotence)
```

Reply:
good poing! thanks, done

Reply:
I meant making it a comment inside the task, not before. Comments outside are usually for multiple tasks.